scripts-utils/daisy202-utils/src/main/resources/xml/xproc/load/load.xpl

```diff
@@ -31,11 +31,40 @@
         <p:pipe port="in-memory.out" step="wrapper"/>
     </p:output>
 
-    <p:import href="http://www.daisy.org/pipeline/modules/common-utils/library.xpl"/>
-    <p:import href="http://www.daisy.org/pipeline/modules/file-utils/library.xpl"/>
-    <p:import href="http://www.daisy.org/pipeline/modules/html-utils/library.xpl"/>
-    <p:import href="http://www.daisy.org/pipeline/modules/fileset-utils/library.xpl"/>
-    <p:import href="http://www.daisy.org/pipeline/modules/mediatype-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/common-utils/library.xpl">
+        <p:documentation>
+            px:message
```

Comment:
out of curiosity, is it used for internal documentation only or for something more advanced?
Looks like a useful piece of info, although can be hard to maintain over time (we need to think about having to update this when the code is changed).

Reply:
It's just internal documentation. Yes it needs updating, but on the other hand it also makes it easier to detect when an import can be removed.

Reply:
ok good!